cluster-autoscaler/simulator/clustersnapshot/predicate/plugin_runner.go

```diff
@@ -111,12 +111,14 @@ func (p *SchedulerPluginRunner) RunFiltersOnNode(pod *apiv1.Pod, nodeName string
 
 	state := schedulerframework.NewCycleState()
 	// Run the PreFilter phase of the framework for the Pod and check the results. See the corresponding comments in RunFiltersUntilPassingNode() for more info.
-	preFilterResult, preFilterStatus, _ := p.fwHandle.Framework.RunPreFilterPlugins(context.TODO(), state, pod)
+	preFilterResult, preFilterStatus, pluginsWithNodes := p.fwHandle.Framework.RunPreFilterPlugins(context.TODO(), state, pod)
```

Comment:
nit: I'd prefer a more accurate name here, `pluginsThatFilteredNodesOut` is too verbose, but maybe something like `filteringPlugins`?

Reply:
sounds good, renamed to `nodeFilteringPlugins`